examples/plot_dataset_example.py

```diff
@@ -0,0 +1,58 @@
+"""
+Simple Moabb Dataset Example
+=========================
+
+Showcasing how to fetch and crop a moabb dataset.
+"""
```

Comment:
this one is rendered fine https://194-232335424-gh.circle-artifacts.com/0/dev/auto_examples/plot_dataset_example.html#sphx-glr-auto-examples-plot-dataset-example-py